<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudienceBidWidget</class>
 <widget class="QWidget" name="AudienceBidWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>266</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{border-image: url(:/image/AudienceBidWidget/bg_custom_jingpai.png);}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="helpBtn">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>7</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image:url(:/image/HostBidUI/bt_help_hover.png);}
QPushButton{border-image: url(:/image/HostBidUI/bt_help.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="girlLb">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>0</y>
      <width>205</width>
      <height>266</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/HostBidUI/img_girl.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="talkLb">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>182</y>
      <width>143</width>
      <height>77</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/HostBidUI/img_talk.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="totalBidTextLb">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>171</y>
      <width>295</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: bold 12px &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;您尚未出价，预计入围价 &lt;span style=&quot; color:#f2c608;&quot;&gt;0&lt;/span&gt; 斗歌豆&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="candidateTextLb">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>134</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 12px &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;已有 &lt;/span&gt;&lt;span style=&quot; font-weight:400; color:#f2c608;&quot;&gt;0&lt;/span&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; 人出价，真英雄，不旁观！&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="songInfoLb">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>100</y>
      <width>309</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fd96fa
</string>
    </property>
    <property name="text">
     <string>歌曲 《songname》    斗歌人数：numPlayer人</string>
    </property>
   </widget>
   <widget class="QLabel" name="douTitleLb">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>70</y>
      <width>299</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/AudienceBidWidget/title_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="countDownWidget" native="true">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>60</y>
      <width>263</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QLabel" name="countDownTextLb">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>263</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/HostBidUI/daoji_text.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="countDownNumLb">
     <property name="geometry">
      <rect>
       <x>223</x>
       <y>3</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26px &quot;微软雅黑&quot;;
color:#00fff6</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bidBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>196</y>
      <width>72</width>
      <height>35</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image:url(:/image/AudienceBidWidget/bt_chujia_hover.png);}
QPushButton{border-image: url(:/image/AudienceBidWidget/bt_chujia.png);
}


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>197</y>
      <width>108</width>
      <height>32</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/AudienceBidWidget/bg_number.png);
color:white;
font: 75 12px &quot;宋体&quot;;</string>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="douPicLb">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>197</y>
      <width>26</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{border-image: url(:/image/AudienceBidWidget/img_beat.png);}

QToolTip{
border-width: 1px;
border-image: url(://image/ToastWidget/toast.png) 2 2 2 2 stretch stretch;
color:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="plusBtn">
    <property name="geometry">
     <rect>
      <x>411</x>
      <y>196</y>
      <width>27</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image:url(:/image/AudienceBidWidget/bt_jia_hover.png);}
QPushButton{border-image: url(:/image/AudienceBidWidget/bt_jia.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minusBtn">
    <property name="geometry">
     <rect>
      <x>411</x>
      <y>214</y>
      <width>27</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image:url(:/image/AudienceBidWidget/bt_jian_hover.png);}
QPushButton{border-image: url(:/image/AudienceBidWidget/bt_jian.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addBidBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>196</y>
      <width>72</width>
      <height>35</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image:url(:/image/AudienceBidWidget/bt_jiajia_hover.png);}
QPushButton{border-image: url(:/image/AudienceBidWidget/bt_jiajia.png);
}


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="talkTextLb">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>200</y>
      <width>127</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,0);
color: white ;
font:12px &quot;宋体&quot;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:12px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:9pt;&quot;&gt;你想好让我做什么了吗？人家有点怕怕的~&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="openLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="aniLabel">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>140</y>
      <width>42</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/AudienceBidWidget/img_jiajia.png);
font:bold 12px &quot;宋体&quot;;
color:white</string>
    </property>
    <property name="text">
     <string>+300</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="minBtn">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>7</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image: 
url(:/image/HostBidUI/bt_min_hover.png);}
QPushButton{border-image: url(:/image/HostBidUI/bt_min.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>7</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image: url(:/image/HostBidUI/bt_close_hover.png);}
QPushButton{border-image: url(:/image/HostBidUI/bt_close.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="miWidget" native="true">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>55</y>
      <width>306</width>
      <height>158</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#miWidget{border-image: url(:/image/AudienceBidWidget/bg_ear.png);}</string>
    </property>
    <widget class="QPushButton" name="okButton">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>101</y>
       <width>165</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{border-image:url(:/image/AudienceBidWidget/bt_done_hover.png);}
QPushButton{border-image: url(:/image//AudienceBidWidget/bt_done.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tipLb">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>22</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;b&gt;&lt;span style=&quot;border: 0px;font-family:'微软雅黑'; font-size:18px; color:#fedc52;&quot;&gt;请插入耳麦后再出价&lt;/span&gt;&lt;/b&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="detailTipLb">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>54</y>
       <width>275</width>
       <height>37</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot;font-family:宋体; font-size:12px; color:#ffffff;&quot;&gt;插入耳麦后，需要点击YY界面上的 &lt;/span&gt;&lt;span style=&quot;font-family:宋体; font-size:12px; color:#26fffd;&quot;&gt;使用新设备&lt;/span&gt;&lt;span style=&quot;font-family:宋体; font-size:12px; color:#ffffff;&quot;&gt; 方&lt;br/&gt;能正常使用奥~&lt;/span&gt; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>194</x>
       <y>101</y>
       <width>91</width>
       <height>39</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{border-image:url(:/image/AudienceBidWidget/bt_cancel_hover.png);}
QPushButton{border-image: url(:/image//AudienceBidWidget/bt_cancel.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
